test/controllers/DetermineEligibilityControllerSpec.scala

```diff
@@ -739,5 +747,34 @@ class DetermineEligibilityControllerSpec extends ItSpec with CombinationsHelper
       redirectLocation(result) shouldBe Some(PageUrls.epayeDebtTooSmallUrl)
     }
 
+    "SA user with IR-SA enrolment but no NINO found, should be directed to generic ineligible page" in {
+      val eligibilityRules = TdAll.eligibleEligibilityRules.copy(part2 = eligibleEligibilityRules.part2.copy(noMtditsaEnrollment = Some(false)))
```

Comment:
in reality we wouldn't get `noMtditsaEnrollment` returned to us in `Ttp.Eligibility.stubRetrieveEligibility`